Trivim/NewApplication.py

```diff
@@ -61,13 +60,15 @@
 from PIL import Image
 ##import canny_main
 import zipfile
-import GUI
+import new
```

Comment:
new is the updated GUI.py file which is why I have requested you to make changes in the GUI.py file  so that we can eliminate importing new.py file in NewApplication.

Reply:
I'll change the name.